### DISCUSSION TITLE
Thinkpad C13 - Incorrect speaker configuration (no sound) on Brunch 126

### DISCUSSION THREAD

**Discussion by djbrad** [sebanc/brunch#2120] [Q&A]
Hi everyone, 

I installed MrChromebox 2405.1 firmware on my Thinkpad C13, as I was always short on space with Chromeos and it felt slow. 

Installed Linux Mint 22 xfce on the emmc, works very well, everything's fine. 

I wanted to keep chromeos on a usb flashdrive, in case I wanted to launch an android app. 

I downloaded the morphius recovery image, and installed with brunch. Chromeos works fine, but I get no sound on the internal speakers (works fine with headphones). 

I tried with the gumboz recovery image, same results. 

I went to console, and I got sound with speaker-test to plughw:1,1 , so I think it's just a wrong audio profile that is loaded. 

I tried a few things, but without success. Does someone got a suggestion?

Thanks a lot!

Brad

**Comment by djbrad** [comment]
Sold the laptop, closing.

**Reply by blchua72** [reply]
I bought a Thinkpad C13 Yoga Chromebook recently, installed the MrChromebox firmware and then installed Windows 10 and then installed Brunch #130 as dual boot. Brunch #130 is working everything except no sound from internal speaker. I found this discussion that match the issue that I am facing. I like to know does anyone still facing  the same issue?  Please share  your experience, suggestion and  solution. Currently I am using headsets that plugs in into the audio jack, then I can have sound from the headset. Bluetooth speaker also works. But it will be nice if the internal speaker can come into life. In Windows 10 everything works Sound, Keyboard, Fan etc, thanks to the drivers from Coolstar.org drivers.

**Reply by blchua72** [reply]
I had made the internal speaker working under brunch 130. 
This is what I have done.
Goto "/usr/share/alsa/ucm" directory,
then "sudo mkdir acp3xalc5682m98357",
then "cd acp3xalc5682m98357.2mic",
then "sudo cp acp3xalc5682m98357.2mic.conf ../acp3xalc5682m98357/acp3xalc5682m98357.conf",
then "sudo cp HiFi.conf ../acp3xalc5682m98357"
then "cd ../acp3xalc5682m98357"
then "ls -l"

you should have the following files:
-----------------------------------------------------------------------------------------------------------------------
chronos@localhost /usr/share/alsa/ucm/acp3xalc5682m98357 $ ls -l
total 8
-rw-r--r--. 1 root root   93 Dec  3 20:35 acp3xalc5682m98357.conf
-rw-r--r--. 1 root root 2983 Dec  3 20:35 HiFi.conf
chronos@localhost /usr/share/alsa/ucm/acp3xalc5682m98357 $ 
------------------------------------------------------------------------------------------------------------------------

Then restart your brunch 130, you should have sound from internal speaker!
